homeassistant/components/control4/manifest.json

```diff
@@ -3,11 +3,10 @@
   "name": "Control4",
   "codeowners": ["@lawtancool"],
   "config_flow": true,
-  "disabled": "This integration is disabled because it uses an old version of socketio.",
   "documentation": "https://www.home-assistant.io/integrations/control4",
   "iot_class": "local_polling",
   "loggers": ["pyControl4"],
-  "requirements": ["pyControl4==1.1.0"],
+  "requirements": ["pyControl4==1.1.3"],
```

Comment:
```suggestion
  "requirements": ["pyControl4==1.2.0"],
```

Reply:
Yep, sorry I didn't see this before I cut the release of pyControl4

Reply:
No worries, it was just a guess anyways.   I wanted to get the PR in the queue so it made it in time for beta